static/app/views/replays/detail/page.tsx

```diff
@@ -46,6 +48,15 @@ export default function Page({
   const onShareReplay = useShareReplayAtTimestamp();
   const onDeleteReplay = useDeleteReplay({replayId: replayRecord?.id, projectSlug});
 
+  const {replay} = useReplayContext();
+  const rrwebFrames = replay?.getRRWebFrames();
+  // The replay data takes a while to load in, which causes our `isVideoReplay`
+  // to return an early `false`, which used to cause UI jumping.
+  // One way to check whether it's finished loading is by checking the length
+  // of the rrweb frames, which should always be > 2 for any given replay.
+  // By default, the 2 frames are replay.start and replay.end
+  const isLoading = !rrwebFrames || (rrwebFrames && rrwebFrames.length <= 2);
```

Comment:
```suggestion
  const isLoading = !rrwebFrames || (rrwebFrames.length <= 2);
```

Reply:
i think we need the `rrwebFrames` check here in case it's `undefined` or `null`

Reply:
Yeah but the `||` should short circuit if undefined or null right?

Reply:
you get a typescript error if you try to access `.length` or something undefined